CLI.md

```diff
@@ -491,39 +491,23 @@ Create genesis configuration for a Linera deployment. Create initial user chains
 
   Default value: `0`
 * `--start-timestamp <START_TIMESTAMP>` — The start timestamp: no blocks can be created before this time
-* `--block-price <BLOCK_PRICE>` — Set the base price for creating a block
+* `--policy-config <POLICY_CONFIG>` — Configure the resource control policy (notably fees) according to pre-defined settings
 
-  Default value: `0`
-* `--fuel-unit-price <FUEL_UNIT_PRICE>` — Set the price per unit of fuel
+  Default value: `default`
 
-  Default value: `0`
-* `--read-operation-price <READ_OPERATION_PRICE>` — Set the price per read operation
+  Possible values: `default`, `only-fuel`, `fuel-and-block`, `all-categories`, `devnet`
```

Comment:
I'm not sure we need all these, and I don't think the names should be user-visible like this. I think we should only have two user-visible ones:
* `testnet` (renamed from `devnet`), and
* `no-fees` (renamed from `default`, where everything is zero and limits are `MAX`).

The other two are, I think, only used in tests, and should be feature-gated?